src/plugins/README.md

```diff
@@ -21,7 +21,7 @@ Extra addons
 ------------
 
 Yet more features are available in presentations that enable 
-[extra addons](../../extras/). Extra addons are 3rd party plugins integrated
+[extra addons](https://github.com/impress/impress-extras). Extra addons are 3rd party plugins integrated
```

Comment:
Thanks for catching this!

In fact, this whole sentence is a bit out of date, it was written at a time when the extra addons were part of the main impress.js repo. While we're doing this, could you please also rewrite the immediately following sentence to something like this:

    Extra addons are 3rd party plugins that are not part of impress.js, but that we have nevertheless collected together into the [impress-extras](https://github.com/impress/impress-extras) repo to provide convenient and standardized access to them. To include the extra addons when checking out impress.js, use `git clone --recursive`. Even then, they are not activated by default in a presentation, rather each must be included with their own `<script>`
tag.

Reply:
Added the new sentence